powerline/segments/i3wm.py

```diff
@@ -1,9 +1,9 @@
 # vim:fileencoding=utf-8:noet
 from __future__ import (unicode_literals, division, absolute_import, print_function)
 
-import i3
-
+import i3ipc
 
+conn = i3ipc.Connection()
```

Comment:
You must not create connection here. Only when actually using `workspaces` segment. And there are too low blank lines here.